docs-src/references/strongly-typed-errors/cause-unspecified.md

```diff
@@ -10,6 +10,6 @@ tags:
 
 This error indicates that the [Workflow Task](/tasks#workflow-task) has failed for an unknown reason.
 
-If you see this error, contact your administrator to file a support ticket or report it [here](https://github.com/temporalio/temporal/issues).
+If you see this error, contact your administrator to [file a support ticket](http://support.temporal.io/) or report it [here](https://github.com/temporalio/temporal/issues).
```

Comment:
We need to remove this, there's no reason to contact an admin or support.

Reply:
Then what should they do?

Reply:
Check the error message and figure out the error in their workflow code.